pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -990,6 +990,23 @@ func applyOutlierDetection(cluster *apiv2.Cluster, outlier *networking.OutlierDe
 		out.EnforcingConsecutive_5Xx = &wrappers.UInt32Value{Value: uint32(0)}             // defaults to 100
 		out.ConsecutiveGatewayFailure = &wrappers.UInt32Value{Value: uint32(outlier.ConsecutiveErrors)}
 	}
+
+	setConsecErrs := func(errs, enforcing **wrappers.UInt32Value, v uint32) {
```

Comment:
why double pointer?

Reply:
cause need to assign value to `errs` and `enforcing`. 
if just pass single pointer, function gets a copy of it. So assigning new value to the given pointer will not result in assigning it to the original one.

Reply:
I would rather just duplicate this logic, this is pretty confusing to me. Its only 4 lines anyways

Reply:
ok, changed it